skyrim-platform/src/platform_se/CMakeLists.txt

```diff
@@ -98,26 +98,25 @@ if(NOT "${SKIP_SKYRIM_PLATFORM_BUILDING}")
 
   include(${cmake_dir}/npm.cmake)
 
-  set(wdir "${CMAKE_CURRENT_BINARY_DIR}/_codegen")
-  file(MAKE_DIRECTORY ${wdir})
-  npm_execute_command(
-    WORKING_DIRECTORY ${wdir}
-    COMMAND init -y
-  )
-  npm_execute_command(
-    WORKING_DIRECTORY ${wdir}
-    COMMAND install tsc-watch@4.2.3 typescript@3.8.3 --save
-  )
-
-  npm_execute_command(
-    WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}/tools/dev_service"
-    COMMAND install
-  )
+   set(wdir "${CMAKE_CURRENT_BINARY_DIR}/_codegen")
```

Comment:
It seems that these lines are not useful at all after porting codegen to C++

Reply:
It seems that these lines are extremely useful to keep the project buildable. That's ok, planning to remove JS stuff in another PR.